src/discord_bot/bot.py

```diff
@@ -5,7 +5,7 @@
 
 
 MODEL_CHOICES = ["gpt-3.5-turbo", "gpt-4", "phi"]
-DEFAULT_MODEL = MODEL_CHOICES[2]
+DEFAULT_MODEL = MODEL_CHOICES[1]
```

Comment:
If we have a default model to run review resume with, does it make sense for the ask llm question command to also have a default model?

Reply:
this seems to be a question for @hungdtrn and @nqngo.

Previously we wanted to offer users multiple models to choose from I think.